core/domain/entities/editor/blocks/widgets/EventAttendees.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace EventEspresso\core\domain\entities\editor\blocks\widgets;
+
+use EventEspresso\core\domain\entities\editor\Block;
+use EventEspresso\core\domain\entities\editor\blocks\CoreBlocksAssetManager;
+use EventEspresso\core\domain\entities\shortcodes\EspressoEventAttendees;
+
+/**
+ * Class EventAttendees
+ * Returns a list of people that have registered for the specified event
+ *
+ * @package EventEspresso\core\domain\entities\editor\blocks\common
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class EventAttendees extends Block
+{
+
+    const BLOCK_TYPE = 'widgets-event-attendees';
+
+    /**
+     * @var EspressoEventAttendees $shortcode
+     */
+    protected $shortcode;
+
+
+    /**
+     * EventAttendees constructor.
+     *
+     * @param CoreBlocksAssetManager $block_asset_manager
+     * @param EspressoEventAttendees $shortcode
+     */
+    public function __construct(CoreBlocksAssetManager $block_asset_manager, EspressoEventAttendees $shortcode)
+    {
+        parent::__construct($block_asset_manager);
+        $this->shortcode = $shortcode;
+    }
+
+
+    /**
+     * Perform any early setup required by the block
+     * including setting the block type and supported post types
+     *
+     * @return void
+     */
+    public function initialize()
+    {
+        $this->setBlockType(self::BLOCK_TYPE);
+        $this->setSupportedPostTypes(array('espresso_events', 'post', 'page'));
```

Comment:
> If this is included with espresso_events post type, is there going to be a way to pass the default event_id for the block (getting it from the loaded event)?
setting the post ID automatically when implemented on an EE_Event CPT will likely have to be done later and I should just remove that for the time being.

> is this going to be filterable at the "exit" point (i.e. where post types are received) so other custom post types can register themselves with this block?
a filter is not really necessary since anyone can simply load the `BlockAssetManagerCollection` via the loader and retrieve any block they want. There's currently a `do_action()`  in `EventEspresso\core\services\editor\BlockRegistrationManager::registerBlocks()` after each block is registered, but it might be more useful to move that to just before each block is registered (or maybe before the loop and then offer the entire collection as a property instead of firing that action repeatedly) so that a block's properties can be modified at the last moment. Another useful place to add an action might be in `EventEspresso\core\services\editor\BlockRegistrationManager::initializeBlocks()` after all blocks are initialized.

Reply:
Yes but as I have said before I **need** to have at least one operational block before I can work on the code for controlling where our blocks get loaded.

Reply:
Is this block operational enough to do this work now?  Is this something that should be done before we merge GBM to Master?

Reply:
> Is this block operational enough to do this work now? Is this something that should be done before we merge GBM to Master?

My question wasn't answered for this block of comments.

Reply:
> My question wasn't answered for this block of comments.

because this question has been already answered many times, both in chat and in code comments

> I need to have at least one operational block

I should have used the same phrasing I have before and said that I'm waiting until there is one block that 
 is completely 100% finished, fully operational, and merged to GBM. I'm not going to risk wasting time trying to figure out why changes to the loading code aren't working if the problem is due to changes to the block itself that is preventing it from loading. Once the block is solidified and there are no more changes in progress, THEN I will finish off the loading code. Nothing to load yet? No loading code.

Reply:
What about the second question,  "is this something that should be done before we merge GBM to master?"  or is the answer something obvious I should have picked up elsewhere (which if I had, I probably wouldn't have asked)?

Reply:
well currently any blocks will just load for any "editor" regardless of the post type, so merging GBM into master just means that block loading won't be conditional. I don't think that will really matter at first simply because I doubt we are going to have any blocks where that will be an issue. At least not for a while.

Reply:
Understood, would it be worthwhile to do an issue so we're tracking that possibility as we add more blocks?  Is that something you'd be willing to do?

Reply:
I think this will be tied very closely if not completely coupled to #485 
My preference from reading through that issue is to go with the first option and have route specific bundles, then we only need to have route identification server side and everything else can just remain as is (we'll see though).